packages/sdk/src/utils/Utils.ts

```diff
@@ -35,20 +35,42 @@ export class Utils {
 
     let cmciResource = `/${CicsCmciConstants.CICS_SYSTEM_MANAGEMENT}/${resourceName}/${cicsPlex}${region}`;
 
-    if (options != null) {
-      if (options.criteria != null && options.criteria.length > 0) {
-        const addParentheses = options.criteria.charAt(0) !== '(';
+    if (options && options.criteria) {
```

Comment:
not required to change this, but if you want to save some typing 😋 

```suggestion
    if (options?.criteria) {
```

This may apply to other if conditions 🙏